gensim/test/test_doc2vec.py

```diff
@@ -30,11 +30,16 @@
 
 
 class DocsLeeCorpus(object):
-    def __init__(self, string_tags=False):
+    def __init__(self, string_tags=False, unicode_tags=False):
         self.string_tags = string_tags
+        self.unicode_tags = unicode_tags
 
     def _tag(self, i):
-        return i if not self.string_tags else '_*%d' % i
+        if self.unicode_tags:
+            return u'_\xa1_%d' % i
```

Comment:
Looks strange, why is it so different with line 41, why you use a `\xa1_'?

Reply:
I enforce a title that is not ASCII encodable, i.e. a unicode character '¡'.

Reply:
@gojomo wdyt?

Reply:
Testing with a non-ASCII character seems sensible/necessary to me!